pkgs/tools/package-management/lix/common.nix

```diff
@@ -149,6 +155,7 @@ stdenv.mkDerivation {
       toml11
       lix-doc
     ]
+    ++ lib.optionals (!isLegacyParser) [ pegtl ]
```

Comment:
nit: i think it should go close to line 131 and maybe be `if else` ?

Reply:
Context is easy to miss in the diff but one is buildInputs and the other nativeBuildInputs.

Reply:
yup exactly.